tests/src/test/scala/com/nvidia/spark/rapids/HashAggregatesSuite.scala

```diff
@@ -120,13 +120,20 @@ class HashAggregatesSuite extends SparkQueryCompareTestSuite {
       assert(cpuPlan.find(_.isInstanceOf[SortAggregateExec]).isDefined)
 
       val gpuPlan = df.queryExecution.executedPlan
+      // execute the plan so that the final adaptive plan is available when AQE is on
+      df.collect()
 
       gpuPlan match {
         case WholeStageCodegenExec(GpuColumnarToRowExec(plan, _)) =>
           assert(plan.children.head.isInstanceOf[GpuHashAggregateExec])
           assert(gpuPlan.find(_.isInstanceOf[SortAggregateExec]).isEmpty)
           assert(gpuPlan.children.forall(exec => exec.isInstanceOf[GpuExec]))
 
+        case a: AdaptiveSparkPlanExec =>
```

Comment:
I think it would be good to revisit creating helper utilities for inspecting plans, that can work both with adaptive and non-adaptive plans so we don't end up with lots of duplicate code in tests. I don't think it has to be part of this PR though.

Reply:
Thanks @andygrove . I have created a follow-on issue for creating helper utilities https://github.com/NVIDIA/spark-rapids/issues/476